wcfsetup/install/files/lib/system/category/CategoryHandler.class.php

```diff
@@ -5,99 +5,97 @@
 use wcf\data\category\Category;
 use wcf\data\object\type\ObjectType;
 use wcf\data\object\type\ObjectTypeCache;
-use wcf\system\cache\builder\CategoryCacheBuilder;
+use wcf\system\cache\eager\CategoryCache;
+use wcf\system\cache\eager\data\CategoryCacheData;
 use wcf\system\exception\SystemException;
 use wcf\system\SingletonFactory;
 
 /**
  * Handles the categories.
  *
- * @author  Matthias Schmidt
- * @copyright   2001-2019 WoltLab GmbH
+ * @author  Olaf Braun, Matthias Schmidt
+ * @copyright   2001-2025 WoltLab GmbH
  * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
  */
-class CategoryHandler extends SingletonFactory
+final class CategoryHandler extends SingletonFactory
```

Comment:
Please remember to change any `protected` to `private` when flagging classes as `final`.